scripts/src/test.sh

```
@@ -41,25 +41,39 @@ workspace_test() {
 }
 
 gsdk_test() {
-  $CARGO test -p gsdk
-  $CARGO test -p gsdk --features vara-testing
+  if [ "$CARGO" = "cargo xwin" ]; then
+    $CARGO test -p gsdk "$@"
```

```suggestion
    $CARGO test -p gsdk --no-fail-fast "$@"
```

---

no fail fast is nextest argument